terratag_test.go

```diff
@@ -170,6 +172,37 @@ func itShouldGenerateExpectedTerratagFiles(suiteDir string, g *GomegaWithT) {
 	}
 }
 
+func getFileSha256(filename string, g *GomegaWithT) string {
+	f, err := os.Open(filename)
+	g.Expect(err).To(BeNil())
+	defer f.Close()
+	h := sha256.New()
+	_, err = io.Copy(h, f)
+	g.Expect(err).To(BeNil())
+
+	return string(h.Sum(nil))
+}
+
+func itShouldGenerateExpectedTerragruntTerratagFiles(entryDir string, g *GomegaWithT) {
+	expectedPattern := entryDir + "/expected/**/*.tf"
+	expectedTerratag, _ := doublestar.Glob(expectedPattern)
+
+	actualTerratag, _ := doublestar.Glob(entryDir + "/out/**/.terragrunt-cache/**/*.tf")
+	actualTerratag = filterSymlink(actualTerratag)
+
+	hashmap := make(map[string]string)
+
+	for _, acctualTerratagFile := range actualTerratag {
+		hashmap[getFileSha256(acctualTerratagFile, g)] = acctualTerratagFile
+	}
+
+	for _, expectedTerratagFile := range expectedTerratag {
```

Comment:
go over the expected output and make there was an "actual" file created with the same sha (for each file).